libstb/drivers/tpm_i2c_infineon.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2013-2016 IBM Corp.
+ * Copyright 2022      3mdeb Sp. z o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __TPM_I2C_INFINEON_H
+#define __TPM_I2C_INFINEON_H
+
+extern void tpm_i2c_infineon_probe(void);
```

Comment:
This doesn't need to be extern. Please remove it here and from Nuvoton.

Reply:
The keyword is superfluous, but the functions are extern, they are explicitly called from `tpm_chip.c`.

Reply:
I think on PPC this could make a difference with LTO (no need for OPD entry), but maybe there are another reasons for it. Since Nuvoton uses it, maybe we can leave it.